### PR TITLE
Phishing v3 - updated quick-view and fixed main layout's attachments section

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
In Progress - need to fix an issue in the playbook and update RN

## Related Issues
fixes: https://github.com/demisto/etc/issues/45811
fixes: https://github.com/demisto/etc/issues/45411

## Description
- Updated the quick-view layout to show "reported" email fields.
- Updated main layout (Investigation) to show correct attachment sections
- Fixed an issue where attachment fields were also set incorrectly (some file information appeared twice)

image

image

## Minimum version of Cortex XSOAR
6.1.0

## Does it break backward compatibility?
 No

### CODE REVIEW

**Comment by idovandijk** [comment]
Ready for review + doc review

**Comment by idovandijk** [comment]
I'm also fixing the attachments section in https://github.com/demisto/etc/issues/45411

File: Packs/Phishing/Playbooks/Process_Email_-_Core_v2.yml [hunk 1]
```diff
@@ -203,7 +192,7 @@ tasks:
       id: 687360e6-7e9e-48bb-80df-7b01a6be74f5
       version: -1
       name: Display email information in layout
-      description: Updates Cortex XSOAR incident fields using data from the email object.
+      description: Updates Demisto incident fields using data from the email object.
```
Comment (tomer-pan) [inline Packs/Phishing/Playbooks/Process_Email_-_Core_v2.yml:195]: Change "Demisto" to "Cortex XSOAR" :)

**Review by tomer-pan — APPROVED** [review]

**Comment by tschanfeld** [comment]
@idovandijk edited the release note file and sent you questions re: the two yaml files via slack.

**Comment by idovandijk** [comment]
> @idovandijk edited the release note file and sent you questions re: the two yaml files via slack.

Since this PR has the `release-notes-only`, everything else should be ignored - they're either minor or automatic changes

**Pull request merged**